src/bin/delete-crate.rs

```diff
@@ -18,6 +18,7 @@ use std::io::prelude::*;
 
 use cargo_registry::Crate;
 
+#[allow(dead_code)]
 fn main() {
```

Comment:
if I replace this with `#[cfg(not(test))]` I'm getting the following soft of errors again:

```
src/bin/delete-version.rs:22:1: 31:2 error: function is never used: `main`, #[deny(dead_code)] on by default
```